docker/backend/cli.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+vendor/bin/sentry-agent
```

Comment:
Wouldn't this run forever, never launching `bin/cake`?

Reply:
It should run forever 🙂

Reply:
Made a change in 21b61b8ea57faf4c1e948f9761a8dd5732eef047